From 50713bede627d1147defbbc6ffe15f27c8f28cb3 Mon Sep 17 00:00:00 2001
From: Cody Xie <cody.xie@rock-chips.com>
Date: Wed, 6 Dec 2017 09:16:26 +0800
Subject: [PATCH] arm: Bump COMMAND_LINE_SIZE to 2048.

The current limit is small for Android Things Verity Boot args.
Bump it.

Change-Id: I091c7f6d4912fec57ecca7dcab38cc99c5b6dfb5
Signed-off-by: Cody Xie <cody.xie@rock-chips.com>
---
 arch/arm/include/uapi/asm/setup.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/include/uapi/asm/setup.h b/arch/arm/include/uapi/asm/setup.h
index 979ff4016404..b4d1bea21a2d 100644
--- a/arch/arm/include/uapi/asm/setup.h
+++ b/arch/arm/include/uapi/asm/setup.h
@@ -16,7 +16,7 @@
 
 #include <linux/types.h>
 
-#define COMMAND_LINE_SIZE 1024
+#define COMMAND_LINE_SIZE 2048
 
 /* The list ends with an ATAG_NONE node. */
 #define ATAG_NONE	0x00000000
-- 
2.35.3

